docker/dev/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1-alpine3.9
+
+WORKDIR /go/src/github.com/unicsmcr/hs_auth
+
+COPY . .
+
+RUN apk add git
```

Comment:
Alpine (the distro the image is based on) doesn't come with git and you need it to download some of the dependencies for the app.

I can probably set it up so that git gets removed after the dependencies are downloaded